src/types/markers.js

```diff
@@ -109,21 +108,23 @@ type GCMajorCompleted_Shared = {
   total_zones: number,
   total_compartments: number,
   minor_gcs: number,
-  store_buffer_overflows: number,
+  // Present when non-zero.
+  store_buffer_overflows?: number,
   slices: number,
 
   // Timing for the SCC sweep phase.
   scc_sweep_total: Milliseconds,
   scc_sweep_max_pause: Milliseconds,
 
   // The reason (if not 'None') why this GC ran non-incrementally.
-  nonincremental_reason: string,
+  nonincremental_reason?: string,
```

Comment:
I understand that only older profiles will have "None". Could you add this in the comment ? And maybe in the type we could have `'None' | string` to make it explicit that this is an expected value for older profiles ?